### PR TITLE
fix: remove microbundle as build tool

### PR DESCRIPTION
**Pull request by illiteratewriter**
- [ ] Bug fix 
- [ ] New feature 
- [ ] Chore 
- [ ] Breaking change 
- [ ] There is an open issue which this change addresses
- [x] I have read the **[CONTRIBUTING](./CONTRIBUTING.md)** document.
- [x] My commits follow the [Git Commit Guidelines](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#commits)
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly.
- [ ] My change requires a change to [Typescript typings](./types/lib).
  - [ ] I have updated the typings accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: scripts/copyDTSFiles.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
```
Comment (davidacevedo) [inline scripts/copyDTSFiles.js:1]: You don't need this files. The type defs are fine where they are.

File: package.json [hunk 2]
```diff
@@ -310,12 +308,12 @@
     "eslint-plugin-jsx-a11y": "^6.5.1",
     "eslint-plugin-react": "^7.30.0",
     "eslint-plugin-storybook": "^0.6.1",
+    "fs-extra": "^11.1.0",
```
Comment (davidacevedo) [inline package.json:311]: Also means you don't need this

**Comment by adarsh-drishya** [comment]
@davidacevedo could you please fastrack the resolution to this issue

**Comment by illiteratewriter** [comment]
closing this because there are some issues with yarn.lock. The child dependencies are getting resolved to versions which does not support Node 12. Has opened a new PR (#2708)

**Pull request closed without merging**